keps/sig-network/20180612-ipv4-ipv6-dual-stack.md

```diff
@@ -613,9 +522,9 @@ Kubeadm will also need to generate dual-stack CIDRs for the --cluster-cidr argum
 This dual-stack proposal will introduce a new IPNetSlice object to spf13.pflag to allow parsing of comma separated CIDRs. Refer to [https://github.com/spf13/pflag/pull/170](https://github.com/spf13/pflag/pull/170)
 
 ### End-to-End Test Support
-End-to-End tests will be updated for Dual Stack. The Dual Stack E2E tests will use deployment scripts from the kubernetes-sigs/kubeadm-dind-cluster github repo to set up a containerized, multi-node Kubernetes cluster that is running in a Prow container (Docker-in-Docker-in-Docker, or DinDinD configuration), similar to the IPv6-only E2E tests (see [test-infra PR # 7529](https://github.com/kubernetes/test-infra/pull/7529)). The DinDinD cluster will be updated to support dual-stack.
+End-to-End tests will be updated for dual-stack. The dual-stack E2E tests will use deployment scripts from the kubernetes-sigs/kubeadm-dind-cluster github repo to set up a containerized, multi-node Kubernetes cluster that is running in a Prow container (Docker-in-Docker-in-Docker, or DinDinD configuration), similar to the IPv6-only E2E tests (see [test-infra PR # 7529](https://github.com/kubernetes/test-infra/pull/7529)). The DinDinD cluster will be updated to support dual-stack.
```

Comment:
we are going to use `kubernetes-sigs/kind` instead of  `kubernetes-sigs/kubeadm-dind-cluster` , we should update it here.
We show replace `(see [test-infra PR # 7529](https://github.com/kubernetes/test-infra/pull/7529))` by https://testgrid.k8s.io/sig-release-master-blocking#kind-ipv6-master-parallel

Reply:
Updated to reflect testing situation. PTAL